README.md

```diff
@@ -14,13 +14,32 @@ TODO
 
 \subsubsection*{Cantor set}
 \subsubsection*{Cartesian space ${\mathbb R}^n$}
+The set of all $n$-tuples, $\{ (x_1, x_2, ... , x_n) : x_i \in \mathbb{R} \}$.\\
+The number $n$ is called the dimension of $\mathbb{R}^n$, and the $n$-tuples
+can be thought of as vectors, since they form a natural vector space.\\
+The set is typically augmented with the "Usual Topology", or sometimes the
+"Euclidean Topology", whose basis is the set of all balls,
+$\{ (x_1, x_2, ..., x_n) : \sqrt{x_1^2 + x_2^2 + ... + x_n^2} < r, r > 0 \}$.
```

Comment:
I think the offsets are missing here - iirc the base is not just arbitrary big balls centered at 0

Reply:
Yeah you're right.